Appraisals

```diff
@@ -0,0 +1,7 @@
+appraise 'rails-5' do
+  source 'https://rubygems.org'
+
+  gem 'rails', '5.1.4'
+  gem 'sqlite3'
```

Comment:
Do we really need database usage (ActiveRecord?) to test requests? I think you can use stubs.

Reply:
Eliminated `sqlite` and `ActiveRecord`